pkg/fanal/analyzer/language/nodejs/license.go

```diff
@@ -0,0 +1,109 @@
+package nodejs
```

Comment:
This is second time I am thinking of moving this file to `nodejs/license` folder, but I'm not sure about it.
@knqyf263 wdyt?

Reply:
sounds good, I'll wait for @knqyf263  answer

Reply:
Sounds good